package.json

```diff
@@ -26,5 +26,34 @@
 	"bugs": {
 		"url": "https://github.com/balena-io/leviathan-worker/issues"
 	},
-	"homepage": "https://github.com/balena-io/leviathan-worker#readme"
+	"homepage": "https://github.com/balena-io/leviathan-worker#readme",
+	"dependencies": {
+		"async-mutex": "^0.1.3",
+		"bluebird": "^3.5.3",
+		"bluebird-retry": "^0.11.0",
+		"etcher-sdk": "^1.0.6",
+		"express": "^4.16.4",
+		"ext2fs": "^1.0.26",
+		"firmata": "^2.0.0",
+		"lodash": "^4.17.11",
+		"mountutils": "^1.3.16",
+		"mz": "^2.7.0",
+		"resin-cli-visuals": "^1.4.2",
+		"resin-image-fs": "^5.0.6"
+	},
+	"devDependencies": {
+		"@types/bluebird": "^3.5.25",
+		"@types/bluebird-retry": "^0.11.2",
+		"@types/express": "^4.16.1",
+		"@types/firmata": "^0.19.0",
+		"@types/lodash": "^4.14.120",
+		"@types/mz": "0.0.32",
+		"@types/node": "^10.12.18",
```

Comment:
Are any of these types used downstream? If so they'll need to go to `dependencies`.